lib/httplib/reverseproxy/reverse_proxy.go

```diff
@@ -119,27 +117,6 @@ func WithFlushInterval(interval time.Duration) Option {
 	}
 }
 
-// WithLogger sets the logger for the forwarder. It uses the logger.Writer()
-// method to get the io.Writer to use for the stdlib logger.
-func WithLogger(logger utils.FieldLoggerWithWriter) Option {
-	return func(rp *Forwarder) {
-		rp.log = logger
```

Comment:
I would keep this assignment as roundTripperWithLogger will use it to log some errors before writing them into response writer with the error handler. These are important to keep so we can track which service originates that debug line. 

This change will make log to be utils.NewLogger()